projects/golang/go/1.15/rpmbuild/SPECS/golang.spec

```diff
@@ -252,6 +252,12 @@ Patch25:       0025-go-1.15.15-eks-path-filepath-do-not-clean-relative-path.patc
 Patch26:       0026-go-1.15.15-eks-update-bundled-golang-org.patch
 Patch27:       0027-go-1.15.15-eks-replace-all-usages-of-BytesOrPanic.patch
 Patch28:       0028-go-1.15.15-eks-multipart-limit-memory-inode-consumption.patch
+Patch29:       0029-go-1.15.15-eks-go-scanner-reject-large-line-a.patch
+Patch30:       0030-go1.15.15-eks-mime-multipart-avoid-excessive.patch
+Patch31:       0031-go1.15.15-eks-net-textproto-mime-multipart-i.patch
+Patch32:       0032-go-1.15.15-eks-mime-multipart-limit-parsed-mi.patch
+Patch105:      0105-add-Cut-method-to-bytes-package.patch
```

Comment:
Patches are applied in this order. Hence, Utility patch is placed before the CVE fix patch.